
float32 max_depth # Maximum tracking depth in meters.
float32 max_track_range # Maximum vertical beam range in meters.